security.md

```diff
@@ -134,7 +134,11 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
-### Physical security
+### Physical Security
+
+* Configure laptops to sleep after 5 minutes you are away from your desk, and require a password to re-open it.

```

Comment:
I'd say "at most 5 minutes" - 5 minutes is quite a long time.

I have my mac set to lock when I move the mouse to the corner of the screen, and I always use it when I leave my desk.